harnessed_jobs/absolute_height_offline/v0/validator_absolute_height_offline.py

```diff
@@ -30,10 +30,11 @@
     zvalues.sort()
     quantiles.sort()
 
-    znom = 12.992  # for ITL
-    zbounds = (-0.009, 0.009)  # from CCD-030
-    quant_low = np.interp(znom - zbounds[0], zvalues, quantiles)
-    quant_high = np.interp(znom - zbounds[1], zvalues, quantiles)
```

Comment:
Apart from the unneeded znom offset and units on zbounds, it looks like the old code was wrong here as well since these old lines have `quant_low >= quant_high`.

Reply:
Yes, it was also wrong.